src/nlsolve/lbroyden.jl

```diff
@@ -51,8 +60,12 @@ end
     Tcache = __lbroyden_threshold_cache(x, threshold)
     @bb mat_cache = copy(x)
 
+    ls_cache = __get_linesearch(alg) === Val(true) ?
```

Comment:
```suggestion
    ls_cache = __use_linesearch(alg) === Val(true) ?
```

Or unify to same name `__get` vs `__use`?

Reply:
Committing to be able to try it out locally!